src/Tokenizer/Tokens.php

```diff
@@ -961,6 +961,10 @@ public function setCode($code)
             $this->registerFoundToken($token);
         }
 
+        if (defined('T_HH_ERROR') && $this->isTokenKindFound(T_HH_ERROR)) {
+            throw new \ParseError('Parsing error, encountered "T_HH_ERROR".');
```

Comment:
Not sure about this error message. If we do keep the token name in the error message, it would be more consistent (I think), to omit the quotes.